Formula/dotnet.rb

```diff
@@ -51,6 +51,18 @@ def install
     (bin/"dotnet").write_env_script libexec/"dotnet", DOTNET_ROOT: libexec
   end
 
+  def caveats
+    <<~EOS
+      To ensure the Homebrew installed version of .NET Core is used, you have to
+      add the following line to your ~/.bash_profile, ~/.zshrc or
+      ~/.config/fish/config.fish file:
+
+        export DOTNET_ROOT="#{opt_libexec}"
+
+      Remember to source the file to update your current session.
```

Comment:
```suggestion
      For other software to find dotnet you may need to set:
        export DOTNET_ROOT="#{opt_libexec}"
```
Let's pattern this after the caveats for keg-only formulae.